src/core/plugins/react-17/index.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @prettier
+ */
+import { render } from "./root-injects"
+
+const React17Plugin = ({ getSystem }) => {
+    const { React } = getSystem()
+    const rootInjects = {}
+
```

Comment:
whole `react-17` plugin can be deleted now as it became the `view-react-17`.

Reply:
Yes, I just saw that. I remove it.